sound/soc/sof/sof-utils.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ */
+
+#ifndef __SOC_SOF_UTILS_H
+#define __SOC_SOF_UTILS_H
+
+struct snd_dma_buffer;
+struct device;
```

Comment:
are these two forward declarations needed? Seems like a missing include in the code that uses sof-utils.h?
@lyakh that one is for you :-)

Reply:
@plbossart thanks :-D No, I find these exactly correct: pointers to these structures are used in this header - in the function prototype below, so, they need to be "announced" (forward-declared). But the internal composition or even size of those structures aren't needed *in this header* so, no need to include respective headers. OTOH, the C-files, that include this header, if they really access members or size of the structures, they will have to include respective headers.

Reply:
I could have added the includes, yes, but I only need these two forward definitions.
The users are including the headers themselves , but  I can do the include here if it is preferred.